.github/workflows/main.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        ref: 'master'
+
+    - name: Running setup
+      if: ${{ matrix.os != 'windows-latest' }}
```

Comment:
Not sure why we exclude here?

Reply:
the command varies slightly for windows and the rest of the platforms.